client/main.lua

```diff
@@ -435,6 +456,27 @@ RegisterNetEvent('lockpicks:UseLockpick', function(isAdvanced)
     end
 end)
 
+AddEventHandler('ox_lib:cache:vehicle', function()
+    showHotwiringLabel()
+end)
+
+
+if config.carjackEnable then
+    AddEventHandler('ox_lib:cache:weapon', function()
+        watchCarjackingAttempts()
+    end)
+end
```

Comment:
Is there a reason/preference to use AddEventHandler vice `lib.onCache`? Also I think that watchCarjackingAttempts thread may end up running multiple times if a player switches weapons versus putting it away and then drawing a new weapon. I haven't done much testing to confirm this though.

Reply:
When you use onCache cache.weapon still is nil. As for the watchCarjackingAttempts, you are right

Reply:
Exactly